package/MDAnalysis/auxiliary/__init__.py

```diff
@@ -0,0 +1,299 @@
+
+If the auxiliary data are more frequent, 
+:meth:`~MDAnalysis.coordinates.base.ProtoReader.next_as_aux` and
+:meth:`~MDAnalysis.coordinates.base.ProtoReader.iter_as_aux` behave the same as :meth:`~MDAnalysis.coordinates.base.ProtoReader.next` and 
+:meth:`~MDAnalysis.coordinates.base.ProtoReader.__iter__`.
+
+
+AuxReader class
+~~~~~~~~~~~~~~~
+
+Registry
+........
+
+In order to facilitate guessing of appropriate AuxReaders, all AuxReaders
+should set as appropriate `format` attribute. For files, this will be the
+expected file extension.
+
+
+Attributes
+..........
+
+The following attributes are inherited from 
+:class:`~MDAnalaysis.auxiliary.base.AuxReader`:
+  ``name``
+      Name under which auxiliary data will be stored in trajectory/Timestep.
+  ``represent_ts_as``
+      Method to use in calculating representative auxiliary value for a 
+      timestep. Default is 'closest'.
+  ``cutoff``
+      Cutoff (in ps) for ignoring auxiliary steps when calculating 
+      representative value(s) for a timestep.
+  ``dt``
+      Change in time between auxiliary steps (in ps). If not specified, will
+      attempt to determine from auxiliary data; otherwise defaults to 1 ps.
+  ``initial_time``
+      Time of first auxiliary step (in ps). If not specified, will attempt to
+      determine from auxiliary data; otherwise defaults to 0 ps.
+  ``time_col``
+      Index of column in auxiliary data storing time, assumed to be in ps
+      (default value ``None``, in which case step time is calculated from ``dt``
+      and ``initial_time``).
+  ``data_cols``
+      List of indices of columns containing data of interest. If not set,
+      will default to all columns excluding that containing time (if present).
+  ``constant_dt``
+      Boolean of whether dt is constant throughout auxiliary data. Default is 
+      ``True``.
+  ``step``
+      Current auxiliary step (0-based).
+  ``n_steps``
+      Total number of auxiliary steps
+  ``n_cols``
+      Number of columns of data for each auxiliary step.
+  ``_data``
+      All recorded data for the current step (split into columns).
+  ``time``
+      Time of current auxiliary step, from the appropriate column of ``_data`` or
+      calculated using ``dt`` and ``initial_time``.
+  ``step_data``
+      List of auxiliary values of interest for the current step, from the 
+      appropriate column(s) of ``_data``.
+  ``ts_data``
+      Numpy array of `step_data` from each auxiliary step assigned to the 
+      last-read trajectory timestep.
+  ``ts_diff``
+      List of difference in time between the last-read trajectory timestep
+      and each auxiliary step assigned to it.
+  ``ts_rep``
+      Represenatative value of auxiliary data for last-read trajectory timestep.
+
+
+:class:`~MDAnalaysis.auxiliary.base.AuxFileReader` additionally provides:
+  ``auxfilename``
+      Name of the auxiliary file.
+  ``auxfile``
+      File object for auxiliary file.
+
+
+Methods
```

Comment:
I think this is mimicking the trajectory API documentation we have.  So this is a declaration of the API, leaving the class docstrings to document the implementation itself?